content/developer/locales/index.md

```diff
@@ -35,6 +35,8 @@ Want to help translate? Awesome! Here are some tips for creating great translati
 
 The [Multilingual plugin](https://open.vanillaforums.com/addon/multilingual-plugin) allows each user select their preference from all enabled locales.
 
+[Subcommunities](/help/multisite/#subcommunities) allow a single site to be split into mutlitple communities. Each community specifies it's own locale.
```

Comment:
I would mention that each subcommunity needs to be associated to a category and the subcommunity takes that locale.